src/lang/english.txt

```diff
@@ -1257,7 +1257,7 @@ STR_CONFIG_SETTING_PLANE_SPEED                                  :Plane speed fac
 STR_CONFIG_SETTING_PLANE_SPEED_HELPTEXT                         :Set the relative speed of planes compared to other vehicle types, to reduce the amount of income of transport by aircraft
 STR_CONFIG_SETTING_PLANE_SPEED_VALUE                            :1 / {COMMA}
 STR_CONFIG_SETTING_PLANE_CRASHES                                :Number of plane crashes: {STRING2}
-STR_CONFIG_SETTING_PLANE_CRASHES_HELPTEXT                       :Set the chance of an aircraft crash happening
+STR_CONFIG_SETTING_PLANE_CRASHES_HELPTEXT                       :Set the chance of a random aircraft crash happening.{}Does not affect plane crashes due to too short runways.
```

Comment:
"to too", while correct, reads weirdly. Perhaps "Does not affect plane crashes on runways that are too short [for the plane]" ?

Reply:
> Does not affect plane crashes on too short runways.

?

Reply:
>Large airplanes can still crash when landing on small airports.